src/BSeries.jl

```diff
@@ -17,6 +17,8 @@ end
 
 using Latexify: Latexify, LaTeXString
 
+using Energy_Preserving
```

Comment:
It's okay to use a sub-module if you want to keep yhe namespace of BSeries.jl clean but not strictly necessary. Anyway, if you want to introduce a new module, please follow the Julia style guide I linked some time ago.